holmes/plugins/toolsets/kafka.py

```diff
@@ -352,17 +382,62 @@ def _invoke(self, params: Dict) -> StructuredToolResult:
             )
 
     def get_parameterized_one_liner(self, params: Dict) -> str:
-        return f"Described topic: {params['topic_name']} in cluster {params['kafka_cluster_name']}"
+        return f"Described topic: {params['topic_name']} in cluster \"{params.get('kafka_cluster_name')}\""
 
 
 def group_has_topic(
-    consumer_group_description: ConsumerGroupDescription, topic_name: str
+    client: AdminClient,
+    consumer_group_description: ConsumerGroupDescription,
+    topic_name: str,
+    bootstrap_servers: str,
 ):
+    # Check active member assignments
     for member in consumer_group_description.members:
-        if len(member.assignment.topic_partitions) > 0:
-            member_topic_name = member.assignment.topic_partitions[0].topic
-            if topic_name == member_topic_name:
+        for topic_partition in member.assignment.topic_partitions:
+            if topic_partition.topic == topic_name:
+                return True
+
+    # Check committed offsets for the topic (handles inactive/empty consumer groups)
+    try:
+        # Try using the Consumer class to check committed offsets for the specific group
```

Comment:
there are a lot of comment here, have you used ClaudeCode? just curios

Reply:
Yes. In this case I kept the comment because it's hard to understand why this code is there without it.
What happens is that is a consumer is not very active (consuming messages slowly) then it was not displayed at all as part of the consumer groups. 
This code changes still return such consumer groups for any given topic along with their state (active or not)